runbotics/runbotics-orchestrator-ui/src/src-app/hooks/useProcessSearch.ts

```diff
@@ -11,15 +11,15 @@ import { processActions } from '#src-app/store/slices/Process';
 
 import useDebounce from './useDebounce';
 
-const DEBOUNCE_TIME = 250;
+const DEBOUNCE_TIME = 400;
 
 const useProcessSearch = (collectionId, pageSize = 12, page = 0) => {
     const searchParams = useSearchParams();
     const router = useRouter();
     const searchFromUrl = searchParams.get('search');
     const searchFieldFromUrl = searchParams.get('searchField');
     const [search, setSearch] = useState(searchFromUrl || '');
-    const [searchField, setSearchField] = useState(searchFieldFromUrl || '');
+    const [searchField, setSearchField] = useState(searchFieldFromUrl || 'name');
```

Comment:
if it's visible for user, than it should be translated
imo if it's searchField value than it should be empty, and there should be a placeholder in the inputs